lesson-proposals/proposed_lessons_20190116.md

```diff
@@ -0,0 +1,10 @@
+These were proposed lessons by Andy Turner as posted in Issue #8
```

Comment:
Filename is wrong I think, I'm guessing you meant `20191016`

Reply:
Yes I did. Good catch!